addons/binding/org.openhab.binding.rfxcom/build.properties

```diff
@@ -0,0 +1,8 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/,\
+               lib/jd2xx.jar
```

Comment:
As you include a 3rd party lib, could you please add an about.html file providing license information about it? See https://eclipse.org/legal/epl/about.php